assets/i18n/qqq.json

```diff
@@ -439,6 +440,7 @@
 	"I18N_PREFERENCES_USERNAME": "Text displayed in the preferences page. - Text shown at the left of the text entry where the user can change his username.\n{{Identical|Username}}",
 	"I18N_PREFERENCES_USERNAME_NOT_SELECTED": "Text displayed in the preferences page. - Text shown in the text entry for the username when there is no username assigned to the user.",
 	"I18N_PROFILE_NO_EXPLORATIONS": "Text displayed on the Profile page. - This message is shown on a user's profile when the user has zero created/edited explorations.",
+	"I18N_REGISTRATION_EXPIRE_MESSAGE": "Message which informs user that the registration session has expired.",
```

Comment:
Ditto re naming of i18n id.

Also follow the format of other descriptions: give some context about where this appears and what its context is.

Reply:
Done.